app/views/staff/shared/xml/_activity.xml.haml

```diff
@@ -1,7 +1,15 @@
 %iati-activity{"default-currency" => activity.default_currency, "xml:lang" => activity.organisation.language_code }
   %iati-identifier= activity.iati_identifier
-  %reporting-org{"type" => activity.organisation.organisation_type, "ref" => activity.organisation.iati_reference }
-    %narrative= activity.organisation.name
+  - service_owner = Organisation.find_by(service_owner: true)
+  - if activity.fund? || activity.programme?
+    %reporting-org{"type" => service_owner.organisation_type, "ref" => service_owner.iati_reference }
+      %narrative= service_owner.name
+  - elsif activity.project? && activity.organisation.name == "UKSA"
```

Comment:
What do you think about changing this to:

`elsif activity.project? && activity.organisation.is_goverment_organisation?`

And implementing that on Organisation? It feels like a small change that works for all current 'goverment' DPs if we do something like:

```
def is_goverment_organisation?
    ["Goverment", "Local Goverment"].include? organisation_type
end
````

Or something?

Reply:
God this is so much more elegant 👌